.github/workflows/test.yml

```diff
@@ -56,6 +56,19 @@ jobs:
           ['tests/f', '4/4'],
           ['tests/k', '1/1']
         ]
+        include:
+          - python-version: 3.7
+            tests: ['tests/f', '1/4']
+            time-zone: 'non-UTC'
+          - python-version: 3.7
+            tests: ['tests/f', '2/4']
+            time-zone: 'non-UTC'
+          - python-version: 3.7
+            tests: ['tests/f', '3/4']
+            time-zone: 'non-UTC'
+          - python-version: 3.7
+            tests: ['tests/f', '4/4']
+            time-zone: 'non-UTC'
```

Comment:
I don't think there is any need to mess with the matrix, we can just run all functional tests non-UTC.

Reply:
Oh ok

Reply:
(I've finished messing around with the matrix now)